locales/en.yml

```diff
@@ -0,0 +1,73 @@
+---
+en:
+  header:
+    logo:
+      image_alt: Logo of Ruby Nord
+    meetup:
+      subscribe: Join the next event
+  home:
+    chat:
+      description_html: |
+        We have a Slack team. We discuss about Ruby news, help each other,
+        talk about next events…
+        Send us an email at %{mailto} to join us.
+      invitation_request_mail: '[ruby-nord] Slack invitation request'
+      join: Join the discussion
+      title: Discuss on our chat
+    description: |
+      We are a group of Ruby and Ruby on Rails developers in North of Paris.
+      We discuss about web technologies (ruby, gems, frameworks…), best
+      practices (agile methods, conventions, tests…) and everything related
+      to Ruby. Whether you're a professional, an advised amateur or just a
+      curious new comer, you're welcome.
+    events:
+      call_to_event: |
+        You think one kind of event is missing. Or maybe there's no event
+        that meets your expectations. Don't hesitate to suggest one.
+      title: Events
+      rails_mentors:
+        description_html: |
+          If you're a new comer to the Ruby / Ruby on Rails world, our
+          <strong>Rails Mentors</strong> workshops and meetings will
+          help you to start your journey in tech.
+        image_alt: Picture of Rails Mentors Lille
+        quote_html: |
+          Help people design, develop and deploy
+          <br/>
+          better web applications, together.
+      ruby_burgers:
+        description_html: |
+          During <strong>Ruby Burgers</strong>, we meet up around a beer and
```

Comment:
I kept the `beer` because it's part of our identity (the North :wink: ) but I added a mention. For sure any other drink is fine too.